### PR TITLE
Added a Alpha note for new users to the readme

### PR DESCRIPTION
**Pull request by EnderNator10**
## Purpose

Warnings to encourage people to test ReactOS in VMS until it becomes more stable.

### CODE REVIEW

**Comment by EnderNator10** [comment]
ps sorry for spelling errors i have dilsexia also i am new to github so dont be to rough to me this is my first pull request(exept svn repos(AROS Project) so i keep it small

**Comment by ThFabba** [comment]
I don't think that's consensus. I would encourage people to try it on hardware. The expectation that it may not work is true for both hardware and VM.

**Comment by Fraizeraust** [comment]
I'd argue the note should be in "More Information" or "What is ReactOS?" section. Besides I do agree with Thomas Faber that ReactOS may or not work at all on hardware or a virtual machine depending on circumstances.

In other words, a correct phrasing would be something like this but of course everyone is welcome to propose a different way:

```ReactOS is currently an Alpha quality software which means the operating system is under heavy development, things may not work well and it can corrupt the data present on the hard disk. It's recommended to test the operating system on a virtual machine or on a computer with no sensitive and credential data!```

**Comment by EnderNator10** [comment]
> 1. **this is not proper markdown (new line before the header is missing)**
> 2. it's a disclaimer not a note
> 3. @Fraizeraust is right: it should be placed under What is ReactOS?
> 4. ETC > etc.
> 5. (TM) > ™

yea like i said i am new to github soexuse the bad markup and i agree with the changes

**Comment by EnderNator10** [comment]
oh i forgot to add to that late one what the update whas

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ ReactOS™ is an Open Source effort to develop a quality operating system that i
 The ReactOS project, although currently focused on Windows Server 2003 compatibility, is always keeping an eye toward compatibility with Windows Vista and future Windows NT releases.
 
 The code of ReactOS is licensed under [GNU GPL 2.0](https://github.com/reactos/reactos/blob/master/COPYING).
-
+***ReactOS is currently an Alpha quality software which means the operating system is under heavy development, things may not work well and it can corrupt the data present on the hard disk. It's recommended to test the operating system on a virtual machine or on a computer with no sensitive and credential data!***
```
Comment (binarymaster) [inline README.md]: Maybe add a linebreak here?

Otherwise LGTM.

**Comment by feikname** [comment]
Nitpick, of course: I believe there should be a comma after "ReactOS is currently an Alpha quality software [..]".

So: "ReactOS is currently an Alpha quality software**,** which means [..]"

**Pull request merged**